### DISCUSSION TITLE
스터디 - 5주차 스터디 목록

### DISCUSSION THREAD

**Discussion by vi2920va** [vi2920va/FE_interview#59] [General]
> 5주차 스터디 목록입니다. 스터디에 주제와 갯수는 강제가 아닌 자율적으로 속도 조절해서 해주시면 됩니다.

## 스터디 일정
목
21:00 ~ 21:15 팀원들 PR 남긴 거 리뷰 시간
21:15 ~ 21:35 모의 면접, 모의면접은 상대방이 공부한 것으로 기준으로 2개 정도 질문하고 피드백 해준다.
21:35 ~ 21:50 모의 면접을 했던 상대방이 아닌 다른 사람에게도 질문하고 싶은 걸 한다.
21:50 ~ 22:00 해당 주차에 회고록 작성, 프론트엔드 질문 정리 질문 했던 댓글 남기기.

## 스터디 모음

송현 
--
- HTML - doctype
- CSS - z-index

남은 
-- 
- JavaScript - call, apply, bind
- React - useReducer vs useState

두현
--
- React-UnitTest

동기
--
- JavaScript - 제너레이터와 async & await

수민
--
- Frontend - Critical Rendering Path
- Frontent - webpack(리팩토링)

영생
--
- Css - em, rem
- FrontEnd - 웹표준과 웹접근성

**Comment by Doodream** [comment]
두현 - React-UnitTest

**Comment by NAMU1105** [comment]
남은 - 5주차 스터디 주제

* JavaScript - call, apply, bind
* React - useReducer vs useState

**Comment by ki7348** [comment]
동기 - JavaScript - 제너레이터와 async & await

**Comment by whdudtod1273** [comment]
영생 - 5주차 스터디 주제
- Css - em, rem
- FrontEnd - 웹표준과 웹접근성

**Comment by ttumzzi** [comment]
수민
- Frontend - Critical Rendering Path
- Frontend - Webpack

**Comment by vi2920va** [comment]
여러분 이번에는 모의면접에 받은 피드백 간단하게 요약하게 작성해주세요.

**Reply by NAMU1105** [reply]
남은 - 마무리를 못하고 나와서 총 피드백은 받지 못했지만, 바인드에 대한 설명을 제대로 하지 못한 것이 아쉽고 useReducer의 동작 설명을 더 추가해야 할 것 같습니다.

**Reply by Doodream** [reply]
유닛 테스트에 대한 기준을 바로잡을 수 있었고 웹팩과 critical rendering path에 대해서 다시한번 복습할 수 있었습니다.

**Reply by vi2920va** [reply]
송현 - Dotype과 웹 표준에 더 공부해야 될거 같습니다. 그래도 팀원들과 많은 이야기를 나누어서 좋았습니다.

**Reply by ki7348** [reply]
동기 - async await과 promise를 더욱 깊게 공부해볼 필요가 있을것 같습니다.

**Reply by ttumzzi** [reply]
수민 - useState와 useReducer의 차이를 다시 복습할 수 있었습니다. CRP에서 세부 내용들을 조금 추가해야할 것 같습니다.

**Reply by whdudtod1273** [reply]
웹 접근성과 웹 표준을 실제 프로젝트에서 더 신경써서 지켜야겠다는 생각을 했습니다.
